### PR TITLE
WIP: Update sunraster docs

### PR DESCRIPTION
**Pull request by DanRyanIrish**
Also address Issues #133 and #134

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+.. _git: https://git-scm.com/
+.. _SunPy Newcomer's Guide: http://docs.sunpy.org/en/stable/dev_guide/newcomers.html
+.. _github guide: https://guides.github.com/
+.. _git guide: https://git-scm.com/book/en/v2/Getting-Started-Git-Basics
+.. _SunPy version control guide: http://docs.sunpy.org/en/stable/dev_guide/version_control.html
+.. _SunPy Developer's Guide: http://docs.sunpy.org/en/stable/dev_guide
+.. _pull requests: https://help.github.com/articles/about-pull-requests/
+.. _Git branches: https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell
+.. _git branch: https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell
+.. _git remote: https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
```
Comment (nabobalis) [inline docs/contributing.rst]: Is it worth adding this?

Comment (DanRyanIrish) [reply docs/contributing.rst]: The link to git remote docs?  Any reason not to?

Comment (nabobalis) [reply docs/contributing.rst]: Sorry, I meant this entire document.

Comment (DanRyanIrish) [reply docs/contributing.rst]: I don’t see a reason not to and it tells people how to report bugs etc. Why do you think it’s better removed?

Comment (nabobalis) [reply docs/contributing.rst]: How/where to report bugs is fine, I mean more the git commands and the like.
The reason we lack them in the sunpy one is due to fact that other places (astropy and github) have some nicer guides that are kept more up to date than what we were doing. I am just worried about the age of this specific document.

**Pull request merged**